Source/Core/Core/CoreTiming.cpp

```diff
@@ -354,7 +355,7 @@ void CoreTimingManager::Throttle(const s64 target_cycle)
   // A maximum fallback is used to prevent the system from sleeping for
   // too long or going full speed in an attempt to catch up to timings.
   const DT max_fallback =
-      std::chrono::duration_cast<DT>(DT_ms(Config::Get(Config::MAIN_TIMING_VARIANCE)));
+      std::chrono::duration_cast<DT>(DT_ms(Config::Get(Config::MAIN_MAX_FALLBACK)));
```

Comment:
This change from 40ms to 100ms causes https://bugs.dolphin-emu.org/issues/13166 (in Dual Core mode).

Now... Dual Core is notorious for being weird, but why exactly was this changed here anyway?

Reply:
having it at 40ms means that 40ms of lag causes slowdown, which stops vi skip from running at full speed if the fps is < 25. I didn't want to make the audio latency 40ms so i didn't increase the variance. i don't know why increasing this would cause it to hang as the throttler is independent of the emulated system, and any changes to the throttler should have no effect on the rest of the code.

Reply:
Remember that Dual Core leaks external state into the emulation. If the larger fallback allows the GPU thread to get further ahead or behind the CPU thread then that's observable by a game in Dual Core mode.

Reply:
is the solution here really to lower the fallback? A higher fallback allows the emulator to run at full speed even during stutters which is a huge bonus.